pylsp/python_lsp.py

```diff
@@ -6,7 +6,7 @@
 import os
 import socketserver
 import threading
-
+import asyncio
```

Comment:
Please move this to be on top of `from functools import partial` and leave the blank line here.